src/ui_framework/doc_site/src/actions/code_viewer/code_viewer_actions.js

```diff
@@ -0,0 +1,30 @@
+
+import ActionTypes from '../action_types';
+
+export default {
```

Comment:
It's generally preferable to expose each value explicitly, rather than an object of named values.

ie.
```js
export const openCodeViewer = slug => ({
  type: ActionTypes.OPEN_CODE_VIEWER,
  slug,
})
```